client/src/main/kotlin/io/provenance/client/grpc/AbstractPbClient.kt

```diff
@@ -115,7 +117,8 @@ open class AbstractPbClient<T : ManagedChannelBuilder<T>>(
     fun broadcastTx(
         baseReq: BaseReq,
         gasEstimate: GasEstimate,
-        mode: ServiceOuterClass.BroadcastMode = ServiceOuterClass.BroadcastMode.BROADCAST_MODE_SYNC
+        mode: ServiceOuterClass.BroadcastMode = ServiceOuterClass.BroadcastMode.BROADCAST_MODE_SYNC,
+        hashHandler: PreBroadcastTxHashHandler? = null,
```

Comment:
nit: probably call variable `txHashHandler` hashHandler maybe confused with hash the currency i guess for :provenance:

Reply:
good call, renamed